Week-7/Exercise7.6.js

```diff
@@ -0,0 +1,24 @@
+let arr = [-1, 2, 1, -4], target = 1;
+arr.sort((a,b) => a-b)
```

Comment:
do not use library functions while solving ds questions, write your own functions